examples/LED/LED.ino

```diff
@@ -40,8 +48,11 @@ void setup(void){
   while (WiFi.status() != WL_CONNECTED) {
     delay(500);
     Serial.print(".");
+    digitalWrite(ledPin, status ? LOW : HIGH); // active low led
+    status = !status;
```

Comment:
Here status can be replaced with a local variable that only takes care of the blinking-while-connecting behavior